src/App.jsx

```diff
@@ -1,33 +1,73 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from "react"
+import "./App.css"
+import EmailItem from "./components/EmailItem"
+import NavBar from "./components/NavBar"
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [emailData, setEmailData] = useState([])
+  const [showNavbar, setShowNavbar] = useState(false)
+
+  useEffect(() => {
+    fetch("https://email-client-api.dev.io-academy.uk/emails")
+      .then((response) => response.json())
+      .then((data) => {
+        setEmailData(data.data)
+      })
+  }, [])
+
+  function handleMenuClick() {
+    if (showNavbar) {
+      setShowNavbar(false)
+    } else {
+      setShowNavbar(true)
+    }
+  }
```

Comment:
OPTIONAL: The code you have written is good and works, but can be expressed more succinctly:

```suggestion
  function handleMenuClick() {
    setShowNavbar(!showNavbar)
  }
```